iati/core/default.py

```diff
@@ -213,16 +213,15 @@ def _populate_schema(schema, version=None):
     Warning:
         Does not create a copy of the provided Schema, instead adding to it directly.
 
-    Todo:
-        Populate the Schema with Rulesets.
-
     """
     version = get_default_version_if_none(version)
 
     codelists_to_add = codelists(version)
     for codelist in codelists_to_add.values():
         schema.codelists.add(codelist)
 
+    schema.rulesets.add(ruleset(version))
```

Comment:
`test_default_schemas_populated` and `test_default_schemas_unpopulated` need modifying to test this.

Reply:
done